pandas/tests/series/test_analytics.py

```diff
@@ -1210,11 +1210,21 @@ def test_idxmin(self):
         result = s.idxmin()
         assert result == 1
 
-    def test_numpy_argmin(self):
-        # argmin is aliased to idxmin
+    def test_numpy_argmin_deprecated(self):
+        # See gh-16830
         data = np.random.randint(0, 11, size=10)
-        result = np.argmin(Series(data))
-        assert result == np.argmin(data)
+
+        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
+            # The deprecation of Series.argmin also causes a deprecation
+            # warning when calling np.argmin. This behavior is temporary
+            # until the implemention of Series.argmin is corrected.
+            result = np.argmin(Series(data))
+            assert result == np.argmin(data)
+
+        with tm.assert_produces_warning(FutureWarning):
+            # argmin is aliased to idxmin
+            result = Series(data).argmin()
```

Comment:
Since you use `Series(data)` twice, let's just write it once and use the variable i.e.
~~~
s = Series(data)
result = s.argmin()
expected = s.idxmin()
assert result == expected
~~~